### DISCUSSION TITLE
V400 update: I am a bit lost... Need Help please !

### DISCUSSION THREAD

**Discussion by alexglvr** [Guilouz/Klipper-Flsun-Speeder-Pad#154] [Q&A]
Hello,

I have a V400. 
I updated it using the Guilouz procedure last year, and while I was making the calibrations, the new official 1.4 firmware was released.
I then updated the Speeder Pad with the V1.4...

I am afraid that it is a complete mess now.
> How can I do to clean all that?
> Does the new released of Guillouz Procedure have the V1.4 features (such a 600mm/sec print speed)?

Sorry if the questions are stupid. But I would like to be sure to understand.

Thank you for your help!

**Comment by Winky1st** [comment]
Hi , im not such an "expert", but what do you think to flash over it v 1.2 as well specified onto the install procedure manual on FAQ?
;) ...cheers and have fun

p.s. by the way you may find this link quite interesting :  https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/discussions/131
